Form/View/ImporterViewType.php

```diff
@@ -0,0 +1,140 @@
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Twig_Environment;
+
+/**
+ * It's the mother class of all specific DataField used in eMS
+ *
+ * @author Mathieu De Keyzer <ems@theus.be>
+ *
+ */
+class ImporterViewType extends ViewType
+{
+
+    /**
+     * @var FileService
+     */
+    private $fileService;
+    /**
+     * @var JobService
+     */
+    private $jobService;
+    /**
+     * @var TokenStorageInterface
+     */
+    private $security;
+    /**
+     * @var Router
+     */
+    private $router;
+
+    public function __construct(FormFactory $formFactory, Twig_Environment $twig, Client $client, LoggerInterface $logger, FileService $fileService, JobService $jobService, TokenStorageInterface $security, Router $router)
+    {
+        parent::__construct($formFactory, $twig, $client, $logger);
+        $this->fileService = $fileService;
+        $this->jobService = $jobService;
+        $this->security = $security;
+        $this->router = $router;
+    }
+
+    public function getLabel()
+    {
+        return "Importer: form to import a zip file containing JSON files";
+    }
+
+    public function getName()
+    {
+        return "Importer";
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+        $builder
+            ->add('finalize', CheckboxType::class, [
+                'data' => true,
+                'required' => false,
+            ])
+            ->add('signData', CheckboxType::class, [
+                'data' => true,
+                'required' => false,
+            ])
+            ->add('businessKey', CheckboxType::class, [
+                'data' => false,
+                'required' => false,
+            ])
+            ->add('rawImport', CheckboxType::class, [
+                'data' => false,
+                'required' => false,
+            ])
+            ->add('force', CheckboxType::class, [
+                'data' => false,
+                'required' => false,
+            ]);
+    }
+
+
+    public function getParameters(View $view, FormFactoryInterface $formFactory, Request $request)
+    {
+        return [];
+    }
+
+    public function generateResponse(View $view, Request $request)
+    {
+        $form = $this->formFactory->create(ImporterType::class, [
+            'view' => $view,
+        ]);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            $fileArray = $form->getData()['archive'];
+            $filename = $this->fileService->getFile($fileArray[EmsFields::CONTENT_FILE_HASH_FIELD] ?? 'missing');
+
+
+            $command = sprintf(
+                "ems:make:document %s %s%s%s%s%s%s",
+                $view->getContentType()->getName(),
+                $filename,
+                $view->getOptions()['rawImport'] ?? false ? ' --raw' : '',
```

Comment:
```suggestion
                $view->getOptions()['rawImport'] ?? '',
```
$view->getOptions()['rawImport']  should return '--raw'.

Same for all lines below!

Reply:
Except that it doesn't return '--raw' it's a checkbox in the option's view. I don't want to save a ' --raw' in the option. If some day the option change.